lib/fluent/plugin/out_sumologic.rb

```diff
@@ -164,6 +164,8 @@ class Fluent::Plugin::Sumologic < Fluent::Plugin::Output
   config_param :retry_min_interval, :time, :default => 1  # 1s
   config_param :retry_max_interval, :time, :default => 5*60  # 5m
 
+  config_param :records_per_request, :integer, :default => 0
```

Comment:
I'd name this `max_records_per_request`. If we wanted parity with the OT exporter, this would be a max request size setting, but that's probably more difficult to do.

Reply:
I rewrite it to be more inline with OTC (except than max_request_size is here a soft limit)